doc/source/tutorial/stats/hypothesis_tests.rst

```diff
@@ -10,5 +10,12 @@ Sample statistics and hypothesis tests
 
 Statistical hypothesis tests are used to decide whether data sufficiently
 support a particular hypothesis. SciPy defines a number of hypothesis tests,
-listed in :ref:`hypotests`. You can find examples to each test in the
-corresponding docstring.
+listed in :ref:`hypotests`.
+
+You can find simple examples to each test in the corresponding docstring. For
```

Comment:
```suggestion
listed in :ref:`hypotests`. You can find simple examples to each test in the corresponding docstring. For
```
Also, I think I'll have some suggestions for the layout of this page when we have some more examples moved over, but it is OK for now.

Reply:
Yeah, this is pretty much a placeholder for now, happy to restructure.